tests/e2e/const.go

```diff
@@ -53,7 +53,7 @@ const (
 	defaultTopicCreationWaitTime         = 10 * time.Second
 	defaultUserCreationWaitTime          = 10 * time.Second
 
-	kafkaClusterCreateTimeout              = 600 * time.Second // Increased from 600 to 700 for multiple kind
+	kafkaClusterCreateTimeout              = 600 * time.Second
 	kafkaClusterResourceCleanupTimeout     = 120 * time.Second
 	kcatDeleetionTimeout                   = 40 * time.Second
 	zookeeperClusterCreateTimeout          = 7 * time.Minute // Increased from 4 to 7 for multiple kind
```

Comment:
Can you please explain why should we remove this?

Reply:
I'm really not sure what sense it makes to have the explanation for this timeout here. For giving information to reviewers it would be fine as a review comment but I don't think this needs to be a comment on the code.

Reply:
+1 that the comment `"// Increased from 4 to 7 for multiple kind"` can be removed since it only provides the context about why the change is made, but it doesn't provide any valuable info itself along with the source code.

Or perhaps we can change the comment to something like this:
```suggestion
        // increase timeout for supporting multiple Kind clusters
	zookeeperClusterCreateTimeout          = 7 * time.Minute
```

Reply:
I can remove but in this case who will face with this issue on multiple kind cluster on github will not know what was the original setting for one kind cluster setup.

Reply:
I think default values and the reason of their changes are valuable in this case.

Reply:
Because the increased timeout was not helped, I put back to the original value and I removed the comment
fixed: [129e7fe](https://github.com/banzaicloud/koperator/pull/1020/commits/129e7fe35b3f9a4d0c70361a85ce70caec6a99a2)